### PR TITLE
Additional rez support on Windows

### LINKED ISSUE #553: Application _LOCATION environment variable being being overridden in rez environment on Windows
@eoyilmaz Added rez supported was added in https://github.com/CGRU/cgru/pull/539, but this only seems to be for Linux.
Currently the `*_LOCATION` environment variable is being overridden in the `cgru/software_setup` scripts. 

For example, when launching Houdini from Keeper that was started from a rez environment I have `HOUDINI_LOCATION` already set. So 

https://github.com/CGRU/cgru/blob/fd58731c4d4a770f2a256a83eb3ca2ed33ed5131/software_setup/setup_houdini.cmd#L4

Should become something like

```cmd
IF NOT DEFINED HOUDINI_LOCATION (
    For /F "Tokens=*" %%I in ('dir /b "C:\Program Files\Side Effects Software\Houdini 1*"') Do set "HOUDINI_LOCATION=C:\Program Files\Side Effects Software\%%I"
)
```

### PR DESCRIPTION
**Pull request by Anti-Distinctlyminty**
Added checks to several setup cmd scripts to prevent `*_LOCATION` variables being overridden.
Changes were only made to applicaitons that I have access to and could test.

Closes #553

### CODE REVIEW

**Comment by timurhai** [comment]
Hi!
Have you checked changes?
Houdini custom command thumbnails now works?

File: software_setup/setup_nuke.cmd [hunk 1]
```diff
@@ -17,19 +17,24 @@ rem SET NUKE_AF_RENDERTHREADS=2
 rem Nuke render launcher:
 SET NUKE_AF_RENDER=nuke
 
-For /F "Tokens=*" %%I in ('dir /b "C:\Program Files\Nuke*"') Do set "NUKE_DIR=C:\Program Files\%%I"
-rem echo %NUKE_DIR%
-For /F "Tokens=*" %%I in ('dir /b "%NUKE_DIR%\Nuke*.*.exe"') Do set "NUKE_EXEC=%NUKE_DIR%\%%I"
+
+IF NOT DEFINED NUKE_LOCATION (
+   For /F "Tokens=*" %%I in ('dir /b "C:\Program Files\Nuke*"') Do set "NUKE_LOCATION=C:\Program Files\%%I"
```
Comment (eoyilmaz) [inline software_setup/setup_nuke.cmd:22]: It is good to have a consistency in the env var nameing, but there might be users depending on the ``NUKE_DIR`` env var. I would suggest also setting this env var.

File: plugins/houdini/afanasy.py [hunk 2]
```diff
@@ -356,6 +356,15 @@ def __init__(self, afnode, ropnode, subblock, prefix, frame_range, for_job_only=
                         cmd_files.evalAsStringAtFrame(self.frame_last)
                     ))
 
+                # Create preview
+                if self.afnode.parm('cmd_preview').eval():
```
Comment (eoyilmaz) [inline plugins/houdini/afanasy.py:360]: Ah okay, so we didn't have previews enabled for custom commands 👍

**Comment by Anti-Distinctlyminty** [comment]
> Hi! Have you checked changes? Houdini custom command thumbnails now works?

Yes. Thumbnails are now generated.

**Comment by eoyilmaz** [comment]
@timurhai I approve the changes 👍

**Comment by timurhai** [comment]
Great!
@Anti-Distinctlyminty thank you for project support!

**Pull request merged**